src/components/Card/Card.js

```diff
@@ -1,6 +1,29 @@
 import './Card.css'
 import createElement from '../../lib/createElement'
 
-export default function Card(name) {
-  return createElement('section', { className: 'Card' }, name)
+export default function Card(image, name, house) {
+  const Card = createElement(
+    'section',
+    { className: 'Card' },
+    image,
+    name,
+    house
+  )
+  Card.style.backgroundColor = getColorByHouse(house)
+  Card.style.color = 'var(--font-color)'
+  Card.innerHTML = `
+  <img class="characterimg" src="${image}">
+  <h2>${name}</h2>
+  <span class="housename">${house}</span>
```

Comment:
Do you use the `housename` class in your css or anywhere? If not, then you could just delete the class. If so, I would also recommend using BEM here.
```suggestion
  <span class="Card__house-name">${house}</span>
```